tests/data_files/jsonschema_f1.json

```diff
@@ -0,0 +1 @@
+{"type":"object","properties":{"abc":{"type":"integer"}}}
```

Comment:
Could you please format this test file and the others so that they are easier to read?